node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeadmin/NodeAdminImpl.java

```diff
@@ -112,9 +91,8 @@ public void refreshContainersToRun(List<NodeSpec> containersToRun) {
         Duration timeBetweenNodeAgents = spread.dividedBy(Math.max(nodeAgentContextsByHostname.size() - 1, 1));
         Instant nextAgentStart = clock.instant();
         // At this point, nodeAgentContextsByHostname and nodeAgentWithSchedulerByHostname should have the same keys
-        for (String hostname : nodeAgentContextsByHostname.keySet()) {
-            NodeAgentContext context = nodeAgentContextsByHostname.get(hostname);
-            nodeAgentWithSchedulerByHostname.get(hostname).scheduleTickWith(context, nextAgentStart);
+        for (Map.Entry<String, NodeAgentContext> entry : nodeAgentContextsByHostname.entrySet()) {
```

Comment:
Consider using `var` the next time

Reply:
I haven't seen many `var`s in our code base. Seems like you're the only user so far :1st_place_medal: